third-party/rsutils/include/rsutils/concurrency/event.h

```diff
@@ -75,11 +78,11 @@ class event
     // If already set, returns immediately
     // Returns true unless a timeout occurred
     template< class Rep, class Period >
-    bool wait( std::chrono::duration< Rep, Period > const & timeout ) const
+    bool wait( std::chrono::duration< Rep, Period > const & timeout )
     {
         std::unique_lock< std::mutex > lock( _m );
         if( ! _is_set )
-            _is_set = ( std::cv_status::timeout != _cv.wait( lock, timeout ) );
+            _is_set = ( std::cv_status::timeout != _cv.wait_for( lock, timeout ) );
```

Comment:
I don't think we should assign `_is_set` here, same way we don't assign in `wait()` function.

Reply:
And then the function can stay `const`

Reply:
Will do so.